packages/core/src/app/shipping/SingleShippingForm.tsx

```diff
@@ -78,6 +79,14 @@ interface SingleShippingFormState {
     hasRequestedShippingOptions: boolean;
 }
 
+function shouldHaveCustomValidation(methodId?: string): boolean {
+    return !(
```

Comment:
For me checking methodId at first, seems easier to read 💅
```
 methodId && !(methodId === PaymentMethodId.BraintreeAcceleratedCheckout || methodId === PaymentMethodId.PayPalCommerceAcceleratedCheckout)
```

Reply:
This is a bit confusing to read, as I see if the methodId is either `PaymentMethodId.BraintreeAcceleratedCheckout` or `PaymentMethodId.PayPalCommerceAcceleratedCheckout` then we should return `false` is that correct?

Reply:
If that is the case can we move the ids to an array.
```
const methodIdsWithoutCustomValidation = [PaymentMethodId.BraintreeAcceleratedCheckout, PaymentMethodId.PayPalCommerceAcceleratedCheckout];

return methodId && !methodIdsWithoutCustomValidation.includes(methodId);
```

Reply:
@animesh1987 yours is better. take a look pls